src/main/java/appeng/tile/misc/TileSecurity.java

```diff
@@ -275,6 +278,41 @@ public int getOwner()
 		return gridProxy.getNode().getPlayerID();
 	}
 
+	/**
+	 * Get a log-readable string representing the owner of this security terminal.
+	 *
+	 * @return string representing the owner
+	 */
+	public String getOwnerName()
```

Comment:
The main reason against putting it in Platform that I had was that this method isn't exactly useful to other code that isn't going directly into display to a human. Code wouldn't want to check if the string was "(no owner)". Perhaps a small data class is more appropriate?